examples/streamlit_app/README.md

```diff
@@ -0,0 +1,72 @@
+# Streamlit Dashboard for Open Source Quartz Solar Forecast
+
+This Streamlit Dashboard provides an interactive interface for running and visualizing solar forecasts using the [Open Source Quartz Solar Forecast](https://github.com/openclimatefix/Open-Source-Quartz-Solar-Forecast) model developed by [Open Climate Fix](https://openclimatefix.org/).
+
+
+https://github.com/user-attachments/assets/940ca4a7-65a4-40ea-a602-ea0f65e96746
+
+
+## Features
+
+- Configure PV site parameters (latitude, longitude, capacity)
+- Select inverter type (No Inverter or Enphase)
+- Enphase API authentication flow (if applicable)
+- Run solar forecast
+- Visualize forecast results with interactive charts
+- Compare forecasts with and without recent PV data (if applicable)
+- Display raw forecast data and provide an option to download it as CSV
+
+## How to Run
+
+1. Clone the repository and install all the dependencies in a virtual environment on a Linux System(or WSL):
+   `pip install -e .` and `pip install -r requirements.txt`
+
+2. Set up environment variables (if applicable):
+
+- Create a `.env` file in your root directory
+- Add the following variables:
+  ```
+  ENPHASE_CLIENT_ID=your_client_id
+  ENPHASE_CLIENT_SECRET=your_client_secret
+  ENPHASE_API_KEY=your_api_key
+  ENPHASE_SYSTEM_ID=your_system_id
```

Comment:
are these optional? Does that dashbaord run if you dont set these?

Reply:
> are these optional? Does that dashbaord run if you dont set these?

Yes they're optional, the forecasts will run even without setting the enphase credentials, in fact it provides the users the option to choose between No Inverter, and Enphase- will be adding more

Reply:
Could you add that in your readme, that these are optional then, please - thanks

Reply:
> Could you add that in your readme, that these are optional then, please - thanks

I've made the changes in my latest commit